ruby-focal/3.2.x-ecs/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM socrata/base-focal:ecs
+MAINTAINER Socrata <sysadmin@socrata.com>
+
+# skip installing gem documentation
+RUN mkdir -p /usr/local/etc \
+	&& { \
+		echo 'install: --no-document'; \
+		echo 'update: --no-document'; \
+	} >> /usr/local/etc/gemrc
+
+ENV RUBY_MAJOR 3.2
+
+# some of ruby's build scripts are written in ruby
+#   we purge system ruby later to make sure our final image uses what we just built
```

Comment:
Lies . . . but I know this is copied and pasted.

Reply:
Might as well start telling the truth now.